src/locale/localed.c

```diff
@@ -430,15 +415,33 @@ static int method_set_vc_keyboard(sd_bus_message *m, void *userdata, sd_bus_erro
                         return sd_bus_error_set_errnof(error, r, "Failed to convert keymap data: %m");
                 }
 
-                /* save the result of conversion to emit changed properties later. */
-                convert = r > 0;
+                x11_modified = r > 0; /* save the result of conversion to emit changed properties later. */
```

Comment:
This causes security issue, as `vconsole_convert_to_x11()` updates `Context`. So, with this change, unprivileged users can update `Context` without any authentications.